### ISSUE TITLE
Null Reference Exception while calling tf.train.AdamOptimizer(learning_rate: learning_rate).minimize(loss)

### ISSUE THREAD

**Issue by wynnhtun** [SciSharp/TensorFlow.NET#527]
This exception was originally thrown at this call stack:
	Tensorflow.Tensor.explicit operator int(Tensorflow.Tensor)
	Tensorflow.Gradients.array_grad._ConcatGradHelper(Tensorflow.Operation, Tensorflow.Tensor, int, int, int)
	Tensorflow.Gradients.array_grad._ConcatGradV2(Tensorflow.Operation, Tensorflow.Tensor[])

Exception Detail:
System.Reflection.TargetInvocationException
  HResult=0x80131604
  Message=Exception has been thrown by the target of an invocation.
  Source=mscorlib
  StackTrace:
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.RuntimeType.InvokeMember(String name, BindingFlags bindingFlags, Binder binder, Object target, Object[] providedArgs, ParameterModifier[] modifiers, CultureInfo culture, String[] namedParams)
   at Tensorflow.ops.<>c__DisplayClass1_1.<RegisterFromAssembly>b__3(Operation oper, Tensor[] out_grads)
   at Tensorflow.gradients_util.<>c__DisplayClass0_2.<_GradientsHelper>b__6(NameScope scope1)
   at Tensorflow.Binding.tf_with[T](T py, Action`1 action)
   at Tensorflow.gradients_util.<>c__DisplayClass0_0.<_GradientsHelper>b__0(NameScope scope)
   at Tensorflow.Binding.tf_with[T](T py, Action`1 action)
   at Tensorflow.gradients_util._GradientsHelper(Tensor[] ys, Tensor[] xs, Tensor[] grad_ys, String name, Boolean colocate_gradients_with_ops, Boolean gate_gradients, Int32 aggregation_method, Tensor[] stop_gradients, Graph src_graph)
   at Tensorflow.Optimizer.compute_gradients(Tensor loss, List`1 var_list, Nullable`1 aggregation_method, GateGradientType gate_gradients, Boolean colocate_gradients_with_ops, Tensor grad_loss)
   at Tensorflow.Optimizer.minimize(Tensor loss, RefVariable global_step, List`1 var_list, GateGradientType gate_gradients, Nullable`1 aggregation_method, Boolean colocate_gradients_with_ops, String name, Tensor grad_loss)
  

Thanks
Regards

**Comment by Oceania2018** [comment]
Please try latest version.
